spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,106 @@
+require_relative '../../app/controllers/restaurants_controller'
+require 'json'
+
+
+describe RestaurantsController, type: :controller do
+  describe "GET index" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe '#create' do
+    it "creates a new restaurant" do
+      post :create, params: { name: 'Artzieli11',
+                                    cuisine: 'pizza',
+                                    rating: 3,
+                                    ten_bis: true,
+                                    address: 'street in tel aviv',
+                                    max_delivery_time: 45 }
+      expect(response.status).to eq(200)
```

Comment:
You say 'create' creates a new restaurant, but you only validate the http status - not the creation of the new restaurant.
From what I see this is something you're doing through the Controller's spec - an HTTP result does not always mean an actual change to some DB entry.